### ISSUE TITLE
Can't use with Cloudflare Pages

### ISSUE THREAD

**Issue by ExEr7um** [pi0/nuxt-shiki#23]
If I deploy to Cloudflare Pages than I get error:
```
500 WebAssembly.instantiate(): Wasm code generation disallowed by embedder
```

I found some information about it in Shiki documentation [here](https://shiki.style/guide/install#cloudflare-workers), but I'm not sure what I should do if I'm using Nuxt module.

**Comment by pi0** [comment]
Thanks for reporting issue. It will be fixed in next release (0.3.0) without need of additional config. In the meantime, you need to set `nitro.experimental.wasm: true`:

```js
export default defineNuxtConfig({
  nitro: {
    experimental: {
      wasm: true,
    },
  },
})

**Comment by ExEr7um** [comment]
Thank you for such a quick response 💚
